### PR TITLE
Adds css for accessibility to set visual focus indication

### PR DESCRIPTION
**Pull request by bretterer**
**Description**
Changes the UX to have a visual indicator when focus is given to a field or button.

**Changes**
 - Adds a box-shadow to the `focused-input` and `link help` classes
 - moves accessibility scss module to the end of the scss list to allow for overrides that deal with accessibility

**Resolves**
 - [OKTA-210992](https://oktainc.atlassian.net/browse/OKTA-210992)

**Photos**

Widget Focus

### CODE REVIEW

File: assets/sass/okta-sign-in.scss [hunk 1]
```diff
@@ -73,13 +72,16 @@
     }
   }
 
+
+  @import 'modules/accessibility';
```
Comment (manueltanzi-okta) [inline assets/sass/okta-sign-in.scss]: Was this moved down intentionally?

Comment (bretterer) [reply assets/sass/okta-sign-in.scss]: Yes, see changes section in the PR description :)

**Comment by bretterer** [comment]
@manueltanzi-okta  in theory, using the two styles that I did, it should work across all screens.  the couple that I tested, it was fine, but I am not sure how to trigger each view we have manually

**Comment by alexdahl-okta** [comment]
FYI @maggielaw-okta, if you'd like to review.

File: assets/sass/modules/_accessibility.scss [hunk 2]
```diff
@@ -8,3 +8,13 @@
   height: 0;
   overflow: hidden;
 }
+
+.focused-input,
```
Comment (edburyenegren-okta) [inline assets/sass/modules/_accessibility.scss]: Any reason not to use `:focus` on the regular input class instead of applying this via JS?

Comment (bretterer) [reply assets/sass/modules/_accessibility.scss]: This was already set up to use `.focused-input`.

I originally started with regular input classes with the `:focus` command, but was very un-reliable when it came to the button.

The other part of this, as the screenshots showed, the checkbox remember me has the full width box, and `:focus` on that element does not work.

**Comment by maggielaw-okta** [comment]
Looks good to me, @bretterer !

**Pull request merged**